wallet-connect/dapp.md

```diff
@@ -0,0 +1,142 @@
+# DApp Integration
+
+Trust uses a forked version of WalletConnect with aditional methods to allow **dApp** developers
+to sign transactions for any blockchain. 
```

Comment:
for any blockchain. - can you link to this url: https://github.com/TrustWallet/wallet-core/blob/master/docs/coins.md ?